packages/react/src/MeasureReportDisplay/MeasureReportDisplay.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, SimpleGrid } from '@mantine/core';
+import { Measure, MeasureReport, MeasureReportGroup, Reference } from '@medplum/fhirtypes';
+import { useMedplum, useResource } from '@medplum/react-hooks';
+import React, { useEffect } from 'react';
+import { MeasureReportDisplayGroup, MeasureTitle } from './MeasureReportGroup/MeasureReportGroup';
+
+export interface MeasureReportDisplayProps {
+  measureReport: MeasureReport | Reference<MeasureReport>;
+}
+
+export function MeasureReportDisplay(props: MeasureReportDisplayProps): JSX.Element | null {
+  const medplum = useMedplum();
+  const report = useResource(props.measureReport);
+  const [measure, setMeasure] = React.useState<Measure | undefined>();
+
+  useEffect(() => {
+    medplum
+      .searchOne('Measure', { url: report?.measure })
+      .then((result: Measure | undefined) => {
+        setMeasure(result);
+      })
+      .catch(console.log);
+  }, [medplum, report]);
+
+  if (!report) {
+    return null;
+  }
+
+  return (
+    <Box>
+      {measure && <MeasureTitle measure={measure} />}
+      <SimpleGrid cols={3} spacing={'xs'}>
```

Comment:
Is there a way to make this responsive for mobile?  3 column makes sense on desktop.  Mobile should probably just be vertically stacked.  I suspect Mantine has a pattern for this.

Reply:
useMediaQuery will always return false for SSR, so made it default 1 col. Can change it back
<img width="676" alt="Screenshot 2023-11-07 at 1 09 09 PM" src="https://github.com/medplum/medplum/assets/6913745/0503038e-ecfc-4a08-8606-b90a1d7f35aa">